test/box/gh-5799-fiber-info.result

```diff
@@ -0,0 +1,73 @@
+-- test-run result file version 2
+test_run = require('test_run').new()
+ | ---
+ | ...
+fiber = require('fiber')
+ | ---
+ | ...
+json = require('json')
+ | ---
+ | ...
+
+function yielder(n) for i=1, n do fiber.yield() end end
+ | ---
+ | ...
+
+csw_check_counter = 0
+ | ---
+ | ...
+fibers = {}
+ | ---
+ | ...
+
+test_run:cmd('setopt delimiter ";"')
+ | ---
+ | - true
+ | ...
+for i=1,100 do
+    fibers[i] = fiber.new(function()
+        for j=1,10 do
+            fiber.yield()
+            if j == fiber.self():csw() and j == fiber.self():info().csw then
+                csw_check_counter = csw_check_counter + 1
+            end
+        end
+    end)
+    fibers[i]:set_joinable(true)
+end;
+ | ---
+ | ...
+for i=1,100 do
+    fibers[i]:join()
+end;
+ | ---
+ | ...
+test_run:cmd('setopt delimiter ""');
+ | ---
+ | - true
+ | ...
+csw_check_counter
+ | ---
+ | - 1000
+ | ...
+
+running = fiber.create(function() fiber.sleep(3) end)
```

Comment:
Please, avoid creating long-living fibers. It might live too long and affect some of the next tests making them flaky. Try to use fiber cond or a busy-yield-loop in the fiber's body to wait for its proper termination before the test ends.

Reply:
Added `fiber.cond()` instead of `fiber.sleep()`.